### ISSUE TITLE
SSH closing problem 

### ISSUE THREAD

**Issue by Alexs-12** [Eugeny/terminus#2825]
Version: 1.0.113
Platform: win32 10.0.19041
Plugins: none

when you want to close the ssh connection terminus does not close the connection and remains blocked

**Comment by ZehuanZhang** [comment]
same the question, even I closed the ssh connection manually through the exit command, terminus still block this connection and window

after that, it must be closed by click the x button and confirm to close ssh connection (sometimes it bothers me because I just want close this windows quickly)

interesting thing is that when the SSH is lost connection unexpected somehow, despite *SSH xxx: session closed* is shown too, terminus will not trying to reconnect while it became connectable

In conclusion, the new logic of ssh reconnection seems working bad than ever 😞 

I guess it will be better to offer a switch in **Settings** to allow users to choose the window behavior even if the SSH reconnection is not perfect now 😆

**Comment by stivo85** [comment]
I can confirm, but the bug occurs when I suce configured ssh connections. If I use _ssh host_ and then disconnect - it's ok.

**Comment by MDKRulez187** [comment]
![Snipaste_2020-08-10_14-25-39](https://user-images.githubusercontent.com/69464221/89782755-27b29580-db16-11ea-8d7d-5c6b74e96ec7.jpg)
Terminus does not end the session cleanly under Linux(Centos), neither with "Exit" nor with "Logout".

**Comment by amirhfarzaneh** [comment]
Same issue here
